sotodlib/preprocess/processes.py

```diff
@@ -1355,6 +1357,28 @@ def save(self, proc_aman, split_flg_aman):
         if self.save_cfgs:
             proc_aman.wrap("split_flags", split_flg_aman)
 
+class UnionFlags(_Preprocess):
+    """Do the union of relevant flags for mapping
+    Typically you would include turnarounds, glitches, etc.
+
+    Saves results for aman under the "flags.glitch_flags" field.
+
+     Example config block::
+
+        - name : "union_flags"
+          process:
+            flag_labels: ['jumps_2pi.jump_flag', 'glitches.glitch_flags', 'turnaround_flags.turnarounds']
+
+    """
+    name = "union_flags"
+    
+    def process(self, aman, proc_aman):
```

Comment:
The union of the flags can be a `calc_and_save` i.e. creation of `total_flags` up to line 1379 then wrapping `total_flags` into `aman.flags.glitch_flags` (along with the added logic I mention in my comment below) can be a `process`